### DISCUSSION TITLE
Ad Hoc targets

### DISCUSSION THREAD

**Discussion by deltabrav0** [warp-tech/warpgate#908] [Q&A]
Is there any way to create and connect to a target in an ad-hoc fashion?  I'm thinking:

https://<warpgate host>:<port>/?warpgate-target=<IP address>

and a connection for the IP would be established.  Maybe no necessary create a persistent target, but at least create the connection.

**Comment by TheBugiMan** [comment]
As an administrator, that ability makes me nervous about allow the server to become an open proxy to anyone with a valid Warpgate user.

If this was to be done, I would want it to be able to be locked down so maybe only particular users able to have 'open access' (e.g. super users) or a 'whitelist' range of IPs.
